### ISSUE TITLE
[Rogue] Tricks bugs out in combat

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#5111]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Whenever you are in combat and spam tricks on your  friendly target it gets bugged (grey out, unable to use,after the buff is gone the CD on tricks never starts,  until you relog) Unforutnetly i can't tell what exactly causes it(it can happen after one use or ten) but i'm pretty sure every rogue can confirm that. It happens in PvE and PvP
**Expected behaviour**: Tell us what should happen instead
Tricks never grey out and you able to use it whenever it is out of CD
**Steps to reproduce the problem**:

1. get yourself a friendly target that is in combat (boss fight for instance)
2. Spam tricks on him
3. Notice how tricks grey out and you unable to use it.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
Happens for MD as well. Just do not spam it as much. This is an issue on every private server and I am guessing it is not that easy to fix since I have not seen a fix anywhere.

**Comment by TeridanDW** [comment]
Hello,
This issue was fixed in #3999 but it caused other problems so it was reverted.
Will reopen the other report.
